deeprvat/deeprvat/train.py

```diff
@@ -438,71 +471,51 @@ def get_min_max_params(self):
 
     def min_max_annotations(self, annotations):
         for key in self.norm_params:
-            min_val = self.norm_params[key]["min"]
-            max_val = self.norm_params[key]["max"]
-            if min_val == max_val:
-                annotations[:, :, key, :] = max_val
-            else: 
-                annotations[:, :, key, :] = (annotations[:, :, key, :] - min_val) / (max_val - min_val)    
+            min = self.norm_params[key]["min"]
+            max = self.norm_params[key]["max"]
```

Comment:
@fmunzlin This change to the normalization function is out-of-date. Keep base branch state.